addons/l10n_es/data/account.account.template-common-canary.csv

```diff
@@ -0,0 +1,11 @@
+id,name,code,user_type_id/id,chart_template_id/id,reconcile
+account_common_canary_4707,"Hacienda Pública, deudor por IGIC",4707,account.data_account_type_current_assets,l10n_es.account_chart_template_common_canary,False
+account_common_canary_47071,"Hacienda Pública, deudora por IGIC régimen transitorio circulante",47071,account.data_account_type_current_assets,l10n_es.account_chart_template_common_canary,False
+account_common_canary_47072,"Hacienda Pública, deudora por IGIC régimen transitorio inversión",47072,account.data_account_type_current_assets,l10n_es.account_chart_template_common_canary,False
+account_common_canary_4727,Hacienda Pública. IGIC Soportado,4727,account.data_account_type_current_assets,l10n_es.account_chart_template_common_canary,False
```

Comment:
As we discussed by mail, I'm still not convinced you need a specific tax account for IGIC, and the given documentation is not specifying that. They only talk about being identified, not being identified by an specific account. In Odoo, taxes are perfectly identified not through accounts, but taxes itself.

Reply:
Different accounts are used to make reconciliation easier, some taxes are paid to the ATC and others to the AEAT.

Reply:
I don't think so. You should have a clearance account like `475000` for Spanish 303, where you move in the regularization entry all the balances to it, but not specific accounts for intermediate tax annotation.

Reply:
Forms 420 (IGIC) and 303 (VAT) are processed by different tax administrations, their review is much easier in separate accounts at the time of a tax inspection.

Reply:
Indeed, simply showing tax report, not ledger by account. You are still using a non Odoo approach...

Reply:
We use Odoo and we understand perfectly well the reasons you mention, what we try to convey are the reasons of the client, an ATC inspection asks for a balance sheet and they want to see IGIC in the account, not IGIC and VAT.

Reply:
As said, for that you don't need a separate account. Canary Islands is not special for that.